packages/manager/modules/vrack/src/partials/available.html

```diff
@@ -17,15 +17,16 @@ <h2 class="vrack__section-title" data-translate="vrack_your_services"></h2>
             data-ng-click="$ctrl.toggleAccordion('available', 'types', $index)"
         >
             <i
-                class="vrack__service-type-icon ovh-font"
+                class="vrack__service-type-icon"
                 data-ng-class="{
-                'ovh-font-dedicatedCloud': serviceType === 'dedicatedCloud',
-                'ovh-font-public-cloud': serviceType === 'cloudProject',
-                'ovh-font-iplb' : serviceType === 'ipLoadbalancing',
-                'ovh-font-ip': serviceType === 'ip',
-                'ovh-font-server': serviceType === 'dedicatedServer' || serviceType === 'dedicatedServerInterface',
-                'ovh-font-network': serviceType === 'dedicatedConnect',
-                'ovh-font-vRack': serviceType === 'legacyVrack'}"
+                'ovh-font ovh-font-dedicatedCloud': serviceType === 'dedicatedCloud',
```

Comment:
At one point, those icons were added to ui-kit https://github.com/ovh/ovh-ui-kit/pull/367, maybe we can check and replace to remove the use of `ovh-manager-webfont`

Reply:
Thanks... But looks like these changes are not on master as yet :)

Reply:
You're right @ganeshkumar1989! These icons were introduced in the `v3.0.0` as Marie mentioned.

But since the `v4.0.0`, they were totally re-organised and some of them has been dropped or replaced.
See: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-content-icons--concept

We can handle the leave `ovh-font-*` in a separate Pull Request I think.

Reply:
Sure @antleblanc . It is better if we handle the icons in a different PR and Jira :)